ReactCommon/cxxreact/SystraceSection.h

```diff
@@ -26,7 +36,7 @@ struct ConcreteSystraceSection {
 public:
   template<typename... ConvertsToStringPiece>
   explicit
-  ConcreteSystraceSection(__unused const char* name, __unused ConvertsToStringPiece&&... args)
+  ConcreteSystraceSection(RCT_UNUSED const char* name, RCT_UNUSED ConvertsToStringPiece&&... args)
```

Comment:
I think we need to change all `__unused` macro in `ReactCommon` if it has portable issue.

Reply:
Oh - I did not notice that it was in other files too! I am trying to just build the `fabric` stuff on MSVC right now, and this is the only `cxxreact` file directly depended on by Fabric (right now).

This tweak should be done to other files, but I would like to keep this pull request small for now - will make it a bit simpler to figure out if the macro I made will have issues when it is merged and ran through Facebook's internal tests